backend/Dockerfile

```diff
@@ -13,24 +28,25 @@ RUN aws configure set default.region ap-southeast-1
 
 WORKDIR /usr/home/postmangovsg
 
-COPY shared ../shared
-RUN cd ../shared && npm ci
+# this means npm will only install production deps, and optimizations in the node
+# engine will take place as well
+ENV NODE_ENV=production 
```

Comment:
shouldnt this step be in the builder phase ?

Reply:
Nope, we need devDependencies in the builder stage to build the app, plus we want this env var set in the final container as well